app/controllers/verify/finance_controller.rb

```diff
@@ -31,7 +31,7 @@ def step_name
     end
 
     def confirm_step_needed
-      redirect_to verify_address_path if idv_session.financials_confirmation == true
+      redirect_to redirect_url if idv_session.financials_confirmation == true
```

Comment:
This change seems unrelated to the feature flag. Can we move this to a separate PR and open a new issue if there isn't one, assuming it's a bug that this always goes to the address path.